kde_ebm/plotting/plotting.py

```diff
@@ -35,14 +36,22 @@ def mixture_model_grid(X, y, mixtures,
 
         hist_dat = [bio_X[bio_y == 0],
                     bio_X[bio_y == 1]]
-
-        hist_c = colors[:2]
+        #* Find useful bin edges for the data: particularly useful for low-dimensional categorical data
+        n_unique_values_bio_X = len(np.unique(bio_X))
+        magic_number = 5
```

Comment:
Constants should be declared at the top of the file and in caps, also this should have a better name.